ckan/cli/generate.py

```diff
@@ -77,3 +80,44 @@ def extension(output_dir):
 
     cookiecutter(template_loc, no_input=True, extra_context=context,
                  output_dir=output_dir)
+
+
+@generate.command(name=u'config', short_help=u'Create config from template.')
+@click.option(u'-o', u'--output-dir', help=u'Location to put the generated '
```

Comment:
I know that the existing `extension` command uses the same pattern, but in my opinion options should be used to pass flags that modify the default behavior. For a parameter that is always required I would use arguments (ie  `@click.argument`). This feels more natural to me than the `-o` flag:

```
ckan generate config /etc/ckan/ckan.ini
```

This will also remove the need for checking if the parameter is provided as I assume click will take care of this.
Also can you change the helper text to "Path to the generated configuration ini file"

Reply:
That is much cleaner. I probably should have asked about that, as I was following the convention in the file, but it seemed that I didn't really need an `option`.